lib/Doctrine/DBAL/Driver/PDOException.php

```diff
@@ -9,10 +9,7 @@
  */
 class PDOException extends AbstractDriverException
 {
-    /**
-     * @param \PDOException $exception The PDO exception to wrap.
-     */
-    public function __construct(\PDOException $exception)
+    public static function fromNativePDOException(\PDOException $exception)
```

Comment:
Note: I had to make this a static method to be able to `throw new PDOException('message')` when I do not have a native `PDOException` object to pass. See for example [this line](https://github.com/doctrine/dbal/pull/3759/files#diff-e5f2acb790dca47e2826055870cfa283R104).